internal/overlord/servstate/handlers.go

```diff
@@ -162,11 +169,48 @@ func (m *ServiceManager) doStart(task *state.Task, tomb *tomb.Tomb) error {
 			}
 			releasePlan()
 		}
-		return fmt.Errorf("cannot start service: exited quickly with code %d", cmd.ProcessState.ExitCode())
+
+		logs, err := getLastLogs(logBuffer)
+		if err != nil {
+			logger.Noticef("Cannot read service %q logs: %v", req.Name, err)
+		}
+
+		msg := fmt.Sprintf("service exited too quickly with code %d", cmd.ProcessState.ExitCode())
```

Comment:
The old message looks slightly more clear. There's no context here about the fact we were attempting to start that service.